[11ty] 2. Cannot find module './LocalAIClient'

```
» yarn build
yarn run v1.22.19
$ eleventy
[11ty] Eleventy CLI Fatal Error: (more in DEBUG output)
[11ty] 1. Error in your Eleventy config file '.eleventy.js'. You may need to run `npm install`. (via EleventyConfigError)
[11ty] 2. Cannot find module './LocalAIClient'
[11ty] Require stack:
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/eleventy-plugin-local-ai/index.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/.eleventy.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/Util/Require.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/TemplateConfig.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/EleventyExtensionMap.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/TemplateData.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/Eleventy.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/cmd.js (via Error)
[11ty]
[11ty] Original error stack trace: Error: Cannot find module './LocalAIClient'
[11ty] Require stack:
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/eleventy-plugin-local-ai/index.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/.eleventy.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/Util/Require.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/TemplateConfig.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/EleventyExtensionMap.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/TemplateData.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/src/Eleventy.js
[11ty] - /Users/doug/code/eleventy-plugin-local-ai-example/node_modules/@11ty/eleventy/cmd.js
[11ty]     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
[11ty]     at Module._load (node:internal/modules/cjs/loader:985:27)
[11ty]     at Module.require (node:internal/modules/cjs/loader:1235:19)
[11ty]     at require (node:internal/modules/helpers:176:18)
[11ty]     at Object.<anonymous> (/Users/doug/code/eleventy-plugin-local-ai-example/node_modules/eleventy-plugin-local-ai/index.js:6:23)
[11ty]     at Module._compile (node:internal/modules/cjs/loader:1376:14)
[11ty]     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
[11ty]     at Module.load (node:internal/modules/cjs/loader:1207:32)
[11ty]     at Module._load (node:internal/modules/cjs/loader:1023:12)
[11ty]     at Module.require (node:internal/modules/cjs/loader:1235:19)
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "eleventy-plugin-local-ai",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "description": "An eleventy plugin for generating static assets from text prompts",
   "main": "index.js",
   "type": "commonjs",
@@ -31,6 +31,7 @@
   },
   "files": [
     "index.js",
+    "LocalAI.js",
     "package.json",
     "README.md"
   ],

